builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,184 +112,206 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
+
+!!! note
+    The respective name for your Client ID variable will vary with the framework you've chosen, e.g., Vite will be `VITE_TEMPLATE_CLIENT_ID`, Next.js will be `NEXT_PUBLIC_TEMPLATE_CLIENT_ID`, and React Native will be `EXPO_PUBLIC_THIRDWEB_CLIENT_ID`.
+
+![thirdweb create API key](/images/builders/ethereum/dev-env/thirdweb/thirdweb-3.webp)
+
 ## Interact With a Contract {: #interact-with-a-contract }
 
-thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/react){target=\_blank}, [React Native](https://portal.thirdweb.com/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
+thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/typescript/v5/react){target=\_blank}, [React Native](https://portal.thirdweb.com/typescript/v5/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript/v5){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
+
+This document will show you how to interact with your contract deployed to Moonbeam using React. You can view the [full React SDK reference](https://portal.thirdweb.com/typescript/v5/react){target=\_blank} in thirdweb’s documentation.
 
-This document will show you how to interact with your contract deployed to Moonbeam using React. You can view the [full React SDK reference](https://portal.thirdweb.com/react){target=\_blank} in thirdweb’s documentation.
+### Run Locally {: #run-locally }
 
-To create a new application pre-configured with thirdweb’s SDKs run and choose your preferred configurations:
+To run your dApp locally for testing and debugging purposes, use the command: 
 
 ```bash
-npx thirdweb create app --evm
+yarn dev
 ```
 
-Or install it into your existing project by running:
+The app will compile and specify the localhost and port number for you to visit in your browser.
 
-```bash
-npx thirdweb install
-```
+![thirdweb run locally](/images/builders/ethereum/dev-env/thirdweb/thirdweb-4.webp)
 
-### Initialize SDK On Moonbeam {: #initialize-sdk-on-moonbeam }
+### Create Client ID {: #create-client-id }
 
-Wrap your application in the `ThirdwebProvider` component and change the `activeChain` to Moonbeam.
+If you generated your thirdweb app with Vite, you'll have a `client.ts` file that looks like the below. As long you've created a `.env` file with your thirdweb API Key (Client ID) defined in `VITE_TEMPLATE_CLIENT_ID`, you can leave the `client.ts` as is and proceed to the next section.
 
-```javascript
-import { ThirdwebProvider } from '@thirdweb-dev/react';
-import { Moonbeam } from '@thirdweb-dev/chains';
+```typescript title="client.ts"
+import { createThirdwebClient } from "thirdweb";
```

Comment:
can you please run prettier on all of the snippets? Noticed these are using double quotes